tests/rt.sh

```diff
@@ -315,7 +315,7 @@ elif [[ $MACHINE_ID = cheyenne.* ]]; then
   PARTITION=
   dprefix=/glade/scratch
   DISKNM=/glade/p/ral/jntp/GMTB/ufs-weather-model/RT
-  STMP=/glade/work
+  STMP=$dprefix
```

Comment:
thanks for making this change, I ran into this issue last week (and just made the change locally).  /glade/scratch is currently scrubbed at 120 days, which should work well for personal-baselines!

Reply:
Yes, I had this issue a few times as well.

In the past, the regression test input data was part of the baseline directory, and it got rsync-ed to maintain the time step (for comparison, syncing to other machines, ...). But this meant that the scrubber sometimes deleted those input data from the regression test baseline directory, causing all sorts of trouble. Therefore the move to $WORK. Now that the input data is kept separately and only the new regression tests results are written to STMP, we don't have this problem anymore.